packages/website/ts/blockchain.ts

```diff
@@ -611,6 +576,45 @@ export class Blockchain {
 
         this._dispatcher.updateBlockchainIsLoaded(true);
     }
+    private async _getInjectedProviderIfExistsAsync(): Promise<InjectedProvider | undefined> {
+        if (!_.isUndefined(this._injectedProviderIfExists)) {
+            return this._injectedProviderIfExists;
+        }
+        let injectedProviderIfExists = (window as any).ethereum;
+        if (!_.isUndefined(injectedProviderIfExists)) {
+            if (!_.isUndefined(injectedProviderIfExists.enable)) {
+                try {
+                    await injectedProviderIfExists.enable();
+                } catch (err) {
+                    errorReporter.report(err);
```

Comment:
do we want to do something nice in the UX if the user declines?

Reply:
We could pop up a toast or something, we have the infrastructure to do it but the design is legacy, from a previous version of portal and doesn't fit in with the current design. I don't want to take up more bandwidth for the team / designers for this I mostly just not want it to completely break when the MetaMask change lands.

Reply:
👍 makes sense